src/client/index.js

```diff
@@ -34,4 +35,20 @@ module.exports.YotiClient = class YotiClient {
   performAmlCheck(amlProfile) {
     return amlService.performAmlCheck(amlProfile, this.pem, this.applicationId);
   }
+
+  /**
+   * Given a dynamic scenario get a custom QR code denoted by the dynamic policy
```

Comment:
Not essential, but this reads a bit easier with a comma:
```suggestion
   * Given a dynamic scenario, get a custom QR code denoted by the dynamic policy
```

Reply:
Fixed in afa86a6